TestApp/app/screens/DataScreen.js

```diff
@@ -27,6 +27,12 @@ export default class DataScreen extends Component {
   // Screen's state.
   state = {
     dataEnabled: false,
+    createDocModalVisible: false,
+    docTtl: 60,
+    docId: "",
```

Comment:
ESLint has 100+ linting errors in the CI https://dev.azure.com/msmobilecenter/SDK/_build/results?buildId=741251.

For example, `Strings must use single quote`.

```suggestion
    docId: '',
```